meson_options.txt

```diff
@@ -6,8 +6,15 @@ option('disable-svml', type: 'boolean', value: 'false',
         description: 'Disable building against SVML')
 option('disable-threading', type: 'boolean', value: 'false',
         description: 'Disable threading support (see `NPY_ALLOW_THREADS` docs)')
-# TODO: flip value to 'false' once we have `npy_cpu_dispatch_config.h` & co.
-option('disable-simd-optimizations', type: 'boolean', value: 'true',
-        description: 'Disable SIMD features beyond the baseline ones')
+option('disable-optimization', type: 'boolean', value: 'false',
+        description: 'Disable CPU optimized code(dispatch,simd,unroll...)')
```

Comment:
Just checking: do we put `optimize("unroll-loops")` behind a flag already now, or is this a change?

Reply:
no, but we should. 

> is this a change

partially, manual unrolls are already behind this flag already: 

https://github.com/numpy/numpy/blob/2289292db6a19f2bbfddd3dea3790ffa19955333/numpy/core/src/umath/loops_minmax.dispatch.c.src#L368-L436